relay-auth/src/lib.rs

```diff
@@ -687,6 +701,104 @@ impl RegisterResponse {
     }
 }
 
+/// A wrapper around a byte vector that represents a signature.
+#[derive(Debug, Clone, PartialEq)]
+pub struct Signature(pub Vec<u8>);
+
+impl Signature {
+    /// Verifies the signature against any of the provided public keys.
+    ///
+    /// Returns `true` if the signature is valid with one of the given
+    /// public keys and satisfies the timestamp constraints defined by `start_time`
+    /// and `max_age`.
+    pub fn verify_signature_any(
```

Comment:
```suggestion
    pub fn verify_any(
```

I think the `signature` is a bit redundant, as it's already on a type called `Signature`.